iceoryx2-bb/posix/src/config.rs

```diff
@@ -118,9 +118,9 @@ pub fn does_system_satisfy_posix_requirements(mode: ComplianceCheckMode) -> bool
     let mut is_compliant: bool = true;
 
     if mode == ComplianceCheckMode::Verbose {
-        println!("{HEADER_COLOR}system requirements check{COLOR_RESET}");
-        println!();
-        println!(" {HEADER_COLOR}minimum system{COLOR_RESET}");
+        eprintln!("{HEADER_COLOR}system requirements check{COLOR_RESET}");
```

Comment:
Everywhere where it checks if `mode == ComplianceCheckMode::Verbose` the console output is explicitly requested by the user and should not be printed to `stderr` but to `stdout`. Could you please revert the changes here in this file.